### PR TITLE
combining two body templates into one for now

### PR DESCRIPTION
**Pull request by 2is10**
@yipingliao cc @andrewconner 

I think this is clearer for now. It makes the differences and similarities in the HTML structure more apparent.

### CODE REVIEW

**Comment by yipingliao** [comment]
:+1:

File: kifi-backend/modules/shoebox/angular/src/layout/body.tpl.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div ng-class="{'public-body': !$root.userLoggedIn}">
+  <header ng-include="$root.userLoggedIn ? 'layout/header/header.tpl.html' : 'layout/header/publicHeader.tpl.html'"></header>
+  <div class="kf-col-island">
+    <aside class="kf-col kf-sidebar" ng-include="'layout/sidebar/sidebar.tpl.html'" ng-if="$root.userLoggedIn"></aside>
+    <section class="kf-scrollable-pane"><!--
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/layout/body.tpl.html:5]: Are the comments necessary? I believe the angular templates compiler does this? (removes whitespace)

**Comment by andrewconner** [comment]
Works for me

File: kifi-backend/modules/shoebox/angular/src/layout/body.tpl.html (hunk 1 above)
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/layout/body.tpl.html:5]: I don’t see why they’d be necessary, but I assumed the funky formatting in the old loggedIn/body.tpl.html was there for a reason and so was playing it safe. Happy to remove them.

**Pull request merged**